libs/platform/src/lib/components/split-menu-button/split-menu-button.component.scss

```diff
@@ -0,0 +1,5 @@
+@import '~fundamental-styles/dist/button-split';
+
+:host {
```

Comment:
Do we need this?button styles are updated as per Fiori3, could you please recheck do we really need it even with latest version of styles